examples/postgres/mockable-todos/README.md

```diff
@@ -0,0 +1,45 @@
+# Mockable TODOs Example
+
+## Description
+
+This example is based on the ideas in [this blog post](https://medium.com/better-programming/structuring-rust-project-for-testability-18207b5d0243). The value here is that the business logic can be unit tested independently from the database layer. Otherwise it is identical to the todos example.
+
+## Setup
+
+1. Declare the database URL
+
+    ```
+    export DATABASE_URL="postgres://postgres:password@localhost/todos"
```

Comment:
Can be nice to provide `.env` and a `docker-compose.yml`, what do you think?

Reply:
I just pushed a commit, does that seem right to you?

Reply:
you dont need to `export` the variable if your app use `dotenv` 😉

Reply:
I thought i would the leave that option in to not stray from what some of the other examples are doing / give people the choice. Thanks for your comments btw 🙂